src/enki_checks.py

```diff
@@ -1,101 +1,36 @@
-    PREFIX_ASSEMBLIES = re.compile(r'.*\/assembly.*\.adoc')
-    PREFIX_MODULES = re.compile(r'.*\/con.*\.adoc|.*\/proc.*\.adoc|.*\/ref.*\.adoc')
-    # should exclude pseudo vanilla like <<some content>>
-    VANILLA_XREF = re.compile(r'<<[^\s]*>>')
-    MULTI_LINE_COMMENT = re.compile(r'(/{4,})(.*\n)*?(/{4,})')
-    SINGLE_LINE_COMMENT = re.compile(r'(?<!\/\/)(?<!\/)^\/\/(?!\/\/).*\n', re.M)
-    EMPTY_LINE_AFTER_ABSTRACT = re.compile(r'\[role="_abstract"]\n(?=\n)')
-    FIRST_PARA = re.compile(r'(?<!\n\n)\[role="_abstract"]\n(?!\n)')
-    NO_EMPTY_LINE_BEFORE_ABSTRACT = re.compile(r'(?<!\n\n)\[role="_abstract"]')
-    COMMENT_AFTER_ABSTRACT = re.compile(r'\[role="_abstract"]\n(?=\//|(/{4,})(.*\n)*?(/{4,}))')
-    VAR_IN_TITLE = re.compile(r'(?<!\=)=\s.*{.*}.*')
-    INLINE_ANCHOR = re.compile(r'=.*\[\[.*\]\]')
-    UI_MACROS = re.compile(r'btn:\[.*\]|menu:.*\[.*\]|kbd:.*\[.*\]')
-    HTML_MARKUP = re.compile(r'(?<!\`|_)<.*>.*<\/.*>|<.*>\n.*\n</.*>(?!\`|_)')
-    INTERNAL_IFDEF = re.compile(r'(ifdef::internal\[\])(.*\n)*?(endif::\[\])')
-    CODE_BLOCK_DASHES = re.compile(r'(-{4,})(.*\n)*?(-{4,})')
-    CODE_BLOCK_DOTS = re.compile(r'(\.{4,})(.*\n)*?(\.{4,})')
-    CODE_BLOCK_TWO_DASHES = re.compile(r'(-{2,})(.*\n)*?(-{2,})')
-    HUMAN_READABLE_LABEL_XREF = re.compile(r'xref:.*\[]')
-    HUMAN_READABLE_LABEL_LINK = re.compile(r'\b(?:https?|file|ftp|irc):\/\/[^\s\[\]<]*\[\]')
-    NESTED_ASSEMBLY = re.compile(r'include.*assembly([a-z|0-9|A-Z|\-|_]+)\.adoc(\[.*\])')
-    NESTED_MODULES = re.compile(r'include.*(proc|con|ref)([a-z|0-9|A-Z|\-|_]+)\.adoc(\[.*\])')
-    RELATED_INFO = re.compile(r'= Related information|\.Related information', re.IGNORECASE)
-    ADDITIONAL_RES = re.compile(r'= Additional resources|\.Additional resources', re.IGNORECASE)
-    ADD_RES_TAG = re.compile(r'\[role="_additional-resources"]')
-    ADD_RES_ASSEMBLY = re.compile(r'== Additional resources', re.IGNORECASE)
-    ADD_RES_MODULE = re.compile(r'\.Additional resources', re.IGNORECASE)
-    EMPTY_LINE_AFTER_ADD_RES_TAG = re.compile(r'\[role="_additional-resources"]\n(?=\n)')
-    COMMENT_AFTER_ADD_RES_TAG = re.compile(r'\[role="_additional-resources"]\n(?=\//|(/{4,})(.*\n)*?(/{4,}))')
-    EMPTY_LINE_AFTER_ADD_RES_HEADER = re.compile(r'== Additional resources\s\n|\.Additional resources\n\n', re.IGNORECASE)
-    CONDITIONAL_IFDEF = re.compile(r'(ifdef|ifndef)::.*\[(?!\s)\]')
-    CONDITIONAL_IFEVAL = re.compile(r'ifeval::.*\[.*?\]')
-    CONDITIONAL_CLOSE = re.compile(r'endif::.*\]')
-    FOOTNOTE_REF = re.compile(r'footnoteref:\[.*?\]')
-
-
-def undetermined_conditional_check(stripped_file):
-    open = re.findall(Regex.OPENING_CONDITIONAL, stripped_file)
-    closed = re.findall(Regex.CLOSING_CONDITIONAL, stripped_file)
-    if len(open) != len(closed):
-        return True
+def too_many_comments_check(original_file, stripped_file, report, file_path):
+    """Checks if more than 1/3 of the file is comments."""
```

Comment:
I am a bit confused about the formula as it looks to me that the code actually reports any file that is less than 0.75 (three quarters) of its original size after removing any comment lines. This directly contradicts the docstring and the message that is then displayed in the report, which claims the comments take more than one third.